src/components/JdsCalendar/Calendar.scss

```diff
@@ -0,0 +1,111 @@
+@use "~/styles/scss/variables" as V;
+
+.jds-calendar {
+  user-select:none;
+  width: 100%;
+  min-height: 120px;
+  border: 1px solid #BDBDBD;
+  display: block;
+  max-width: 224px;
+  border-radius: 6px;
+  z-index: 20;
+  padding: 12px;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 23px;
+  font-family: 'Roboto';
+  box-shadow: 0 4px 10px 0 rgb(0,0,0,.08);
+
+  &__days{
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+
+    &-day {
+      cursor: pointer;
+      width: 32px;
+      height: 32px;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      font-family: 'Lato';
+      color: V.$gray-700;
+      border-radius: 50%;
+
+      &:hover{
+        background: V.$green-700;
+        color: white;
+      }
+
+      &--selected{
+        background: V.$green-700;
+        color: white !important;
+      }
+
+      &--today{
+        font-weight: bold;
+        color: V.$gray-900;
+      }
+
+      &--sunday {
+        color: V.$red-700;
```

Comment:
i think `--holiday` would give a better context for future usage?

Reply:
Alright, i will change that ASAP